### PR TITLE
(PA-3176) Bump ruby to 2.5.8

### PR DESCRIPTION
**Pull request by Dorin-Pleava**
This PR bumps ruby component to 2.5.8.

### CODE REVIEW

**Comment by Dorin-Pleava** [comment]
Travis CI build failed because travis also builds agent-runtime-6.4.x and ruby the ruby-2.5.7.rb file is no longer available.

**Comment by Dorin-Pleava** [comment]
Should be merged after https://github.com/puppetlabs/puppet-runtime/pull/318.

**Comment by GabrielNagy** [comment]
@Dorin-Pleava can you post some passing builds/ad-hoc runs?

**Comment by Dorin-Pleava** [comment]
Packaging for runtime: https://jenkins-master-prod-1.delivery.puppetlabs.net/view/vanagon-generic-builder/job/platform_vanagon-generic-builder_vanagon-packaging_generic-builder/781/
Ad-hoc for puppet-agent that uses the runtime: https://jenkins-master-prod-1.delivery.puppetlabs.net/view/Adhoc/job/platform_puppet-agent-extra_puppet-agent-packaging_adhoc-ad_hoc/918/

File: configs/components/ruby-2.5.8.rb [hunk 1]
```diff
@@ -217,7 +217,7 @@
     pkg.install do
       [
         "#{host_ruby} ../rbconfig-update.rb \"#{rbconfig_changes.to_s.gsub('"', '\"')}\" #{rbconfig_topdir}",
-        "cp original_rbconfig.rb #{settings[:datadir]}/doc/rbconfig-2.5.7-orig.rb",
+        "cp original_rbconfig.rb #{settings[:datadir]}/doc/rbconfig-2.5.8-orig.rb",
```
Comment (ciprianbadescu) [inline configs/components/ruby-2.5.8.rb:220]:
```suggestion
        "cp original_rbconfig.rb #{settings[:datadir]}/doc/rbconfig-#{pkg.get_version}-orig.rb",
```

**Comment by mihaibuzgau** [comment]
@puppetlabs/bolt, @scotje, @puppetlabs/skeletor this passed Puppet Agent pipeline, we're planning to get it merged tomorrow. Please let us know if you've got any concerns.

**Comment by donoghuc** [comment]
No concerns from Skeletor. Thanks!

**Comment by lucywyman** [comment]
+1 from Bolt

**Pull request merged**